_posts/2023-06-30-july-10-downtime.md

```diff
@@ -1,12 +1,13 @@
 ---
 title: Extended Downtime July 10th - 14th, 2023
 state: active
-severity: 1
+severity: 2
```

Comment:
why are we increasing the severity?

Reply:
thank you, good catch, that was a typo, meant to be 0